bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md

```diff
@@ -0,0 +1,51 @@
+* **Topic: review of https://github.com/bitcoin/bitcoin/pull/27601**
```

Comment:
```suggestion
---
title: Discussion on open Coin Selection matters
categories: ['core-dev-tech']
tags: ['bitcoin core', 'coinselection']
date: 2023-09-19
---

* **Topic: review of https://github.com/bitcoin/bitcoin/pull/27601**
```

Not sure the date is correct. I think it was Tuesday that we talked?

Reply:
Yes, it was Tuesday. Accepting the suggestion, will squash later when not on mobile